src/main/java/subsystems/elbow/Elbow.java

```diff
@@ -0,0 +1,47 @@
+package subsystems.elbow;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Elbow extends GBSubsystem {
+    private static Elbow instance;
+    private static CANSparkMax motor;
+
+    private Elbow() {
+        motor = new CANSparkMax(ElbowConstants.ELBOW_MOTOR_ID, ElbowConstants.ELBOW_MOTOR_TYPE);
```

Comment:
Again mention that the type is brushless 
![image](https://github.com/user-attachments/assets/ffd9a7d9-b528-4e85-85fc-5b66cfbcd8c5)